src/script/ir_builder/relax/ir.cc

```diff
@@ -320,6 +305,54 @@ TVM_REGISTER_GLOBAL("script.ir_builder.relax.If").set_body_typed(If);
 TVM_REGISTER_GLOBAL("script.ir_builder.relax.Then").set_body_typed(Then);
 TVM_REGISTER_GLOBAL("script.ir_builder.relax.Else").set_body_typed(Else);
 
+//////////////////////// Symbolic Shape Rewriter ////////////////////////
+
+using tvm::relax::Expr;
+class SymbolicShapeRewriter : public tvm::relax::StructInfoMutator {
+ public:
+  explicit SymbolicShapeRewriter(Map<String, tvm::tir::Var> var_table)
+      : var_table_(std::move(var_table)) {}
+
+  Array<tvm::tir::Var> undefined_vars_;
```

Comment:
Public field names are not appended with underscores.
```suggestion
  Array<tvm::tir::Var> undefined_vars;
```

Reply:
In this particular case given it is internal use(within this class), I think it is fine to also keep the underscore as it is